src/checkout-buttons/strategies/masterpass/masterpass-button-strategy.ts

```diff
@@ -53,15 +53,21 @@ export default class MasterpassButtonStrategy implements CheckoutButtonStrategy
 
     private _createSignInButton(containerId: string): HTMLElement {
         const buttonContainer = document.getElementById(containerId);
+        const state = this._store.getState();
+        const paymentMethod = this._methodId ? state.paymentMethods.getPaymentMethod(this._methodId) : null;
 
         if (!buttonContainer) {
             throw new Error('Need a container to place the button');
         }
 
+        if (!paymentMethod) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
         const button = document.createElement('input');
 
         button.type = 'image';
-        button.src = 'https://static.masterpass.com/dyn/img/btn/global/mp_chk_btn_160x037px.svg';
+        button.src = `https://${paymentMethod.config.testMode ? 'sandbox.' : ''}src.mastercard.com/assets/img/btn/src_chk_btn_126x030px.svg?locale=en_us&paymentmethod=master,visa,amex,discover&checkoutid=${paymentMethod.initializationData.checkoutId}`;
```

Comment:
Are we sure we want to hardcode the locale here? No strong objection really, I just find it interesting that you can pass the locale in (which seems to make a difference - i.e. `nl_nl`: ![nl_nl](https://src.mastercard.com/assets/img/btn/src_chk_btn_126x030px.svg?locale=nl_nl&paymentmethod=master,visa,amex,discover&checkoutid=123) vs `en_en`: ![nl_nl](https://src.mastercard.com/assets/img/btn/src_chk_btn_126x030px.svg?locale=en_us&paymentmethod=master,visa,amex,discover&checkoutid=123)), but we don't